diesel_cli/src/database.rs

```diff
@@ -157,17 +157,20 @@ pub fn reset_database(
     migrations_dir: &Path,
 ) -> Result<(), crate::errors::Error> {
     drop_database(&database_url(args)?)?;
-    setup_database(args, migrations_dir)
+    setup_database(args, migrations_dir, false)
```

Comment:
This changes the current default behavior. We should also use the --no-default-migration flag here instead.

Reply:
Done.